### PR TITLE
[Site-wide] Update the outdated links to the sign-in-faq page

### PR DESCRIPTION
**Pull request by ncksllvn**
## Description
`/sign-in-faq` has been replaced with R&S pages, so this PR updates the FE links.

https://github.com/department-of-veterans-affairs/va.gov-team/issues/15782
## Testing done
Looked at the sign-in modal

## Screenshots
This change to the sign in modal is the only visual change I think

image

## Acceptance criteria
- [ ] Links point to correct locations

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

**Review by sanlouise — APPROVED** [review]

File: src/platform/site-wide/cta-widget/components/messages/mvi/NotAuthorized.jsx [hunk 1]
```diff
@@ -22,7 +22,7 @@ function NotAuthorized({ authenticatedWithSSOe }) {
           Verify Your Identity
         </button>
         <p>
-          <a href="/sign-in-faq/#how-to-verify">
+          <a href="/resources/verifying-your-identity-on-vagov/">
```
Comment (ncksllvn) [inline src/platform/site-wide/cta-widget/components/messages/mvi/NotAuthorized.jsx:25]: This is shown inside the call-to-action alert boxes on health care pages if the user isn't verified

File: src/platform/user/authentication/components/SignInModal.jsx [hunk 2]
```diff
@@ -295,9 +298,18 @@ export class SignInModal extends React.Component {
...
+                  signing in
+                </a>{' '}
+                and{' '}
+                <a
+                  href="/resources/verifying-your-identity-on-va-gov/"
+                  target="_blank"
+                >
+                  verifying your identity
                 </a>
+                .
```
Comment (ncksllvn) [inline src/platform/user/authentication/components/SignInModal.jsx:312]: This whole thing is the sign-in modal

**Comment by ncksllvn** [comment]
@department-of-veterans-affairs/vsa-public-websites-frontend can someone review this plz?

**Review by zacharymorel — APPROVED** [review]
LGTM

**Review by kelsonic — APPROVED** [review]

**Review by theodur — APPROVED** [review]

**Pull request merged**